### PR TITLE
Use Ubuntu 20.04

### PR DESCRIPTION
**Pull request by mikemonteith**
Ubuntu 16.04 is not supported by Azure DevOps any more

### CODE REVIEW

File: templates/deployment-job.yml [hunk 1]
```diff
@@ -9,19 +9,18 @@ jobs:
 - job: validateParameters
   displayName: 'Validate template parameters'
   steps:
-    - bash: $(System.DefaultWorkingDirectory)/scripts/validate-template-parameters.sh
+    - bash: $(System.DefaultWorkingDirectory)/scripts/validate-template-parameters.sh '$(REVIEW_APP_NAME_ROOT)$(System.PullRequest.PullRequestId)'
```
Comment (mikemonteith) [inline templates/deployment-job.yml:12]: This bash script doesn't take an argument

**Comment by mikemonteith** [comment]
If you want to change the keyvault stuff, you need to update the README.md

**Comment by mikemonteith** [comment]
If you want to change the keyvault stuff, you need to update the README.md

File: templates/deployment-job.yml (hunk 1 above)
Comment (mayank1211) [reply templates/deployment-job.yml]: Correct, removed the parameter and probably can either remove or refactor this task to check all the secrets and variables fetched from keyvault. (Probably task for later on).

**Review by mayank1211 — APPROVED** [review]
I am merging the changes for today's deployment and will create another PR to change/remove and refactor the code.

**Pull request merged**